config/src/config/test_data/validator.config.yaml

```diff
@@ -0,0 +1,57 @@
+base:
+    data_dir: "/opt/libra/data/common"
+    role: "validator"
+    waypoint:
+        type: "from_storage"
```

Comment:
[For later] Same `type` comment as waypoint.

Reply:
I think this might be possible in yaml